src/components/BTextArea/BTextArea.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <div :class="classes">
+    <label v-if="!hideLabel" :for="id">
+      <strong> 
+        {{ label }}<span v-if="required" aria-label="Required">*</span>
+      </strong>   
+    </label>
+    <textarea
+      v-model="inputValue"
+      v-bind="{id, name, cols, rows, placeholder, required, disabled}"
+      :placeholder="placeholder ? placeholder : null"
```

Comment:
If placeholder is null and you put it in the v-bind object with the rest, wouldn't it be the same?